content/services/csv/powerdev_open_source_projects.csv

```diff
@@ -60,6 +60,7 @@
 "RPM Fusion","A community driven third party repository for Fedora and Enterprise Linux. They use a dedicated VM as a builder connected to their main infrastructure for building packages for ppc64le platform"
 "SHA-3","Ensures support and viability of Secure Hash Algorithm 3 on POWER, including POWER8 specific optimizations"
 "`Snowpatch`_","An open source tool which provides CI testing for open-source projects which use a traditional patches + email system for accepting contributions; improves the state of continuous integration for a number of open source projects which are critical to the OpenPOWER community, providing automated test capabilities, similar to work already being done on the x86 architecture"
+"Swift", "Turing complete programming language for Unix-based operating systems. Supports objects-oriented, protocol-oriented, and functional programming paradigms."
```

Comment:
"Turing complete"? I think you'll be hard pressed to find a major programming language that isn't turing complete, a non-turing complete programming language is much harder to make than a turing-complete language, and it's not very useful in anything outside of proofs.

What I'm saying is that this term is way too broad in describing a programming language, since nearly every major programming language is turing complete. Is there a better, more precise way of describing this programming language?